code401cgallenges/ArrayReverse.java

```diff
@@ -0,0 +1,41 @@
+import java.util.Arrays;
+
+// import com.sun.org.apache.xpath.internal.SourceTree;
+
+public class ArrayReverse {
+  public static void main(String[] args) {
+
+    int[] myArray = { 1, 2, 3, 4, 5 };
+    reverseArray(myArray, myArray.length);
+
+    int[] testArray = {6, 7, 8, 9};
+    anotherReverseArray(testArray, testArray.length);
+
+  }
+
+  // Creating new Array
+  public static void reverseArray(int[] arr, int n) {
+    int[] newArray = new int[n];
+    int j = n;
```

Comment:
Instead of passing in `n`, couldn't you have just gotten the length of the int[] being passed in?